modules/cloudwatch_logzio/setup.tf

```diff
@@ -1,15 +0,0 @@
-provider "aws" {
-  version = "2.70.0"
-  region  = var.aws_region
-  profile = var.aws_profile
-}
-
-provider "archive" {
```

Comment:
Was this archive completely unused?

Reply:
I'm not sure. In this `cloudwatch_logzio` module we're still using `data.archive_file` but I don't see any other modules in this repo contain a `provider` blocks.